tensorflow_datasets/text/stanford_quad.py

```diff
@@ -0,0 +1,212 @@
+# coding=utf-8
```

Comment:
as we already have squad.py (that has 1.1) version - consider changing name of this file (and the class) to Squad2.0

Reply:
Done @cyfra